src/aspect/event/ExcreteEPSCumulative.java

```diff
@@ -39,6 +39,15 @@ public class ExcreteEPSCumulative extends Event
 	public String SPECIES = XmlRef.species;
 	public String SEARCH_DIST = XmlRef.epsDist;
 	
+	
+	/**
```

Comment:
The reasoning is that EPS can come in multiple forms (closely associated EPS capsule and loosely associated simulated here as separate particles). When the agent reaches the EPS threshold it loses part of its capsule EPS, not all of it. Though I agree this could use further customization as this would be good as a user-defined value with a backup default value. In iDynoMiCS this value is an uniRand from .6 to .9 (not settable by user).

Reply:
I've created an issue for this for later improvement #216